### PR TITLE
Fix iframe switching functional tests

### PR DESCRIPTION
**Pull request by AndreyBelym**

### CODE REVIEW

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

File: src/client/driver/driver.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ export default class Driver {
         if (this.skipJsErrors || this.contextStorage.getItem(TEST_DONE_SENT_FLAG))
             return Promise.resolve();
 
-        var error = new UncaughtErrorOnPage(err.msg || err.message, err.pageUrl);
+        var error = new UncaughtErrorOnPage(err.stack || err.msg || err.message, err.pageUrl);
```
Comment (inikulin) [inline src/client/driver/driver.js]: Why we need `err.stack` here?

Comment (AndreyBelym) [inline src/client/driver/driver.js]: Just trying to debug. BTW it's useless, because IE9 causes most troubles here.

**Comment by AndreyBelym** [comment]
It's still WIP now.

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by AndreyBelym** [comment]
I was able to reproduce issue with hanging tests on IE9 on my own VM. Looks like the `iframe-driver` sometimes fails to initialize, and not sends 'ready' messages to the server, judging by IE DevTools Network Log.

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
Seems like build bot died. Is there anyone except @AlexanderMoskovkin, who can restart it?

**Comment by inikulin** [comment]
Nevermind, bot was stopped by @churkin

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by AndreyBelym** [comment]
Okay, \cc @inikulin. Honestly, I do not know what exactly is going on in IE9, and thehack is ugly, but at least we need to increase test timeout.

File: src/client/driver/iframe-driver.js [hunk 2]
```diff
@@ -70,6 +70,9 @@ export default class IframeDriver extends Driver {
 
                 if (inCommandExecution)
                     this._onReady(new DriverStatus({ isCommandResult: true }));
+            })
+            .then(() => {
+                // HACK: For IE9
```
Comment (inikulin) [inline src/client/driver/iframe-driver.js]: I believe we just need next tick here. Returning `delay(0)` in the previous promise should do a trick.

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
We can do `.then(() => delay(0))`

**Comment by AndreyBelym** [comment]
The first failure are caused by Saucelabs VM, the second by `overlapped drag` unstable test, because I forgot to rebase my PR.

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
@testcafe-build-bot \retest

**Comment by inikulin** [comment]
lgtm

**Pull request merged**